shardingsphere-infra/shardingsphere-infra-context/src/main/java/org/apache/shardingsphere/infra/context/metadata/MetaDataContextsBuilder.java

```diff
@@ -37,6 +37,7 @@
 import org.apache.shardingsphere.infra.optimize.context.OptimizeContextFactory;
 import org.apache.shardingsphere.infra.rule.ShardingSphereRule;
 import org.apache.shardingsphere.infra.rule.builder.ShardingSphereRulesBuilder;
+import org.apache.shardingsphere.infra.rule.builder.ShardingSphereRulesBuilderMaterials;
```

Comment:
materials? or material?

Reply:
> materials? or material?

@tristaZero `ShardingSphereRulesBuilderMaterials` is named with reference to `SchemaBuilderMaterials`. Should the style be consistent?